src/main/resources/com/google/api/codegen/ruby/main.snip

```diff
@@ -435,10 +454,22 @@
   {@context.upperCamelToLowerUnderscore(method.getSimpleName())}
 @end
 
-@private makeApiCall(method, request, isReturnEmptyMessage)
-  @@{@getMethodName(method)}.call({@request}, options)
-  @if isReturnEmptyMessage
-    nil
+@private makeApiCall(method, service, request, isReturnEmptyMessage)
```

Comment:
I feel that we can pass 'methodConfig' together with 'method' and 'request' instead.

We can use methodConfig.isLongRunningOperation(), methodConfig.getLongrunningConfig(), methodConfig.isReturnEmptyMessage inside 'makeApiCall'. In this way, we don't need to depend on context, which will make migrating to MVVM easier.